src/Extensions/SwiftTypeCrawlerExtension.php

```diff
@@ -0,0 +1,313 @@
+<?php
+
+namespace Ichaber\SSSwiftype\Extensions;
+
+use Ichaber\SSSwiftype\Service\SwiftypeCrawler;
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Control\Director;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Versioned\Versioned;
+
+/**
+ * @property DataObject|$this $owner
+ */
+class SwiftTypeCrawlerExtension extends DataExtension
```

Comment:
`Swiftype` rather than `SwiftType`.

I also don't think we want anyone applying this extension to their classes? If that's the case, then this should be `abstract`. This forces folks to pick either the `File` or `SiteTree` extensions specifically.

Reply:
Tbh this has been my main problem with swiftype over the years, can't seem to spell it correctly!